templates/deployment.yaml

```diff
@@ -630,7 +630,40 @@ spec:
         - name: pcap-data
           mountPath: /data/
       {{- end }}
-
+      {{- if .Values.rclone.enabled }}
+      ## Rclone ##
+      - name: rclone
+        image: {{ .Values.rclone.image.repository }}:{{ .Values.rclone.image.tag }}
+        volumeMounts:
+        - name: pcap-data
+          mountPath: /data/
+        imagePullPolicy: {{ .Values.rclone.image.pullPolicy | quote }}
+        ports:
+        - containerPort: 22
+          name: ssh-sftp
+        resources:
+{{ toYaml .Values.resources.rclone | indent 10 }}
+        securityContext:
+        capabilities:
+          add:
+          - NET_ADMIN
+        envFrom:
+          - configMapRef:
+              name: {{ template "CGW.fullname" . }}-rclone-env
+        command:
+        - sh
+        - -c
+        - |
+           mkdir data/finished
+           watchnames=''
+           [ -d /data/ ] && watchnames="$watchnames /data/"
```

Comment:
Can you please explain what is happening here?

Reply:
please read https://www.tldp.org/LDP/Bash-Beginners-Guide/html/ … it explains test-command ('[', ']', '-d'), variable substitution etc.